doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -92,7 +92,7 @@ In order for the Kubernetes components to be conformant with the Reference Archi
 |`ra2.k8s.008`|System Resource Reservations|To avoid resource starvation issues on nodes, reserve compute resources for system daemons and Kubernetes system daemons such as kubelet, container runtime, etc. (requires Kubernetes version 1.17 or later). Use the following kubelet flags: `--reserved-cpus=[a-z]`|TBC||
 |`ra2.k8s.009`|CPU Pinning|When hosting workloads matching the Network Intensive profile, in order to support CPU Pinning, the kubelet must be started with the `--cpu-manager-policy=static` option. (Note, only containers in `Guaranteed` pods - where CPU resource `requests` and `limits` are identical - and configured with positive-integer CPU `requests` will take advantage of this. All other Pods will run on CPUs in the remaining shared pool.)|[infra.com.cfg.003](./chapter02.md#223-cloud-infrastructure-software-profile-requirements)||
 |`ra2.k8s.010`|IPv6DualStack|To support IPv6 and IPv4, the `IPv6DualStack` feature gate must be enabled on various components (requires Kubernetes v1.16 or later). kube-apiserver: `--feature-gates="IPv6DualStack=true"`. kube-controller-manager: `--feature-gates="IPv6DualStack=true" --cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR> --service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR> --node-cidr-mask-size-ipv4 ¦ --node-cidr-mask-size-ipv6` defaults to /24 for IPv4 and /64 for IPv6. kubelet: `--feature-gates="IPv6DualStack=true"`. kube-proxy: `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR> --feature-gates="IPv6DualStack=true"`|[req.inf.ntw.04](./chapter02.md#23-kubernetes-architecture-requirements)||
-|`ra2.k8s.011`|||||
+|`ra2.k8s.011`| Anuket profile labels | To clearly identify which worker nodes are compliant with the different profiles defined by CNTT the worker nodes must be labelled according to the following pattern: an `anuket.io/profile/basic` label must be set to `true` on the worker node if it can fullfill the requirements of the [basic profile](../../../ref_model/chapters/chapter04.md#4.2.5) and an `anuket.io/profile/network-intensive` label must be set to `true` on the worker node if it can fullfill the requirements of the [network intensive profile](../../../ref_model/chapters/chapter04.md#4.2.5).|||
```

Comment:
For consistency
```suggestion
|`ra2.k8s.011`| Anuket profile labels | To clearly identify which worker nodes are compliant with the different profiles defined by Anuket the worker nodes must be labelled according to the following pattern: an `anuket.io/profile/basic` label must be set to `true` on the worker node if it can fullfill the requirements of the [basic profile](../../../ref_model/chapters/chapter04.md#4.2.5) and an `anuket.io/profile/network-intensive` label must be set to `true` on the worker node if it can fullfill the requirements of the [network intensive profile](../../../ref_model/chapters/chapter04.md#4.2.5).|||
```

Reply:
@CsatariGergely could you update this and then we can merge, thanks!